pkg/commands/process/settings/rules/javascript/aws_lambda/query_injection/testdata/ok_no_injection.js

```diff
@@ -0,0 +1,18 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { QueryCommand } from "@aws-sdk/client-dynamodb";
+
+const dynamodb = new DynamoDBClient({ region: "af-south-1" })
+
+exports.handler = async function(event, _context) {
+  const params = {
+    KeyConditionExpression: "Title = " + getTitle(),
+    FilterExpression: "contains (Author, :name)",
+    ExpressionAttributeValues: {
+      ":name": { S: getAuthorName() },
+    },
+    ProjectionExpression: "Title, Author",
+    TableName: "BOOKS_TABLE",
+  };
```

Comment:
It's safe to reference the event object in this kind of hash but I couldn't get the pattern to work for it. Anyone any ideas 😅

Reply:
^ After discussion with @cfabianski we're fine to fail on these cases, as it is still user-defined input coming into the query